swri_transform_util/src/wgs84_transformer.cpp

```diff
@@ -110,7 +110,12 @@ namespace swri_transform_util
     if (local_xy_util_->Initialized())
     {
       std::string local_xy_frame = local_xy_util_->Frame();
-      if (tf_listener_->frameExists(local_xy_frame))
+      if (!tf_listener_)
+      {
+        ROS_WARN("Cannot initialize a Wgs84Transformer with an invalid transform listener.");
+        initialized_ = false;
```

Comment:
I'm not sure if changing the initialized_ value here is a good,

Reply:
Yep, I see now that its just going to get overwritten by the return value anyway. I do think the state should `initialized_ == false` if the tf listener is not valid, but I will change that to a local variable that gets returned.